scss/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "scss",
```

Comment:
[REQ][TIP]: Name in package.json should contain project name :) will introduce to node and npm with javascript  at monday.